src/styles/utils/_variables.scss

```
@@ -0,0 +1,14 @@
+$day-color: #eee;
+$day-size: 100px;
+$day-gap: 1px;
+$row-pad: 10px;
+$month-width: 7 * $day-size + 6 * $day-gap + 2 * $row-pad;
+$week-days: (
+  "start-day--mon": 0,
+  "start-day--tue": 1,
```

Take the modifier name from scss nesting.
```suggestion
  "mon": 0,
  "tue": 1,
```